packages/shared/providers.yaml

```diff
@@ -1123,7 +1125,7 @@ gong-oauth:
     disable_pkce: true
     token_request_auth_method: basic
     proxy:
-        base_url: https://api.gong.io
+        base_url: ${connectionConfig.api_base_url_for_customer} || https://api.gong.io
```

Comment:
connectionConfig seems to be camelCase, I assume we can't rewrite properties in `token_response_metadata`?

Reply:
Yes currently we can't rewrite these properties.